src/parser.rs

```diff
@@ -260,29 +202,20 @@ fn test_get_read_u32_fn() {
 
 #[test]
 fn test_parse_catalog() {
-    macro_rules! assert_variant {
-        ($value:expr, $variant:path) => {
-            match $value {
-                $variant => (),
-                _ => panic!("Expected {:?}, got {:?}", $variant, $value),
-            }
-        };
-    }
-
     let fluff = [0; 24]; // zeros to pad our magic test cases to satisfy the length requirements
 
     {
         let mut reader = vec![1u8, 2, 3];
         reader.extend(fluff.iter().cloned());
         let err = parse_catalog(&reader[..], ParseOptions::new()).unwrap_err();
-        assert_variant!(err, Eof);
+        matches!(err, Eof);
```

Comment:
Oops, I forgot to put it in `assert!()`.

> Perhaps we should keep assert_variant for now? thinking

About bumping the versions, I see the good in both options, I'll let you decide what is the best, I can revert those changes if you ask me to.

I personally don't care too much about it, but maybe it is because of my target user base (for the project I'm currently focusing on, where I expect people to use it in rolling-release distros).

Reply:
> @justinas, I'm waiting for a response for the `matches!` replacement, I think that you'd want me to revert the changes, but I'm not sure, I'll wait a little more, I think that I have 1 week left for counting this to HacktoberFest
> 
> EDIT: I'm also willing to separating the commits as you suggested me :D.

@marcospb19, I would say, let's be conservative and revert the `matches!` thing. Travis CI should pass having done that.